### ISSUE TITLE
Dependencies for compiling and generic library

### ISSUE THREAD

**Issue by rea987** [MrAlert/sdlcl#18]
As I have made a fresh Ubuntu 18.04 installation, I am sure about dependencies for compiling for 32 bit applications in 64 bit system. Also, is it possible to make a generic library which would work on most of mainstream distros?

**Comment by rea987** [comment]
@strycore Yeah, that seems to work fine on Ubuntu Mate 18.04, thanks a lot! Though, the library still has the ETQW specific bug which needs fixing; https://github.com/MrAlert/sdlcl/issues/14#issuecomment-388630899

**Comment by rea987** [comment]
@strycore Will you compile generic 64 bit version of the library for few 64 bit SDL1.2 games such as Unreal Tournament 2004?

**Comment by strycore** [comment]
@rea987 sure, there it is: https://lutris.net/files/tools/sdlcl-1.0-x86_64.tar.gz
